hardware/esp8266com/esp8266/libraries/Wire/Wire.cpp

```diff
@@ -45,9 +45,6 @@ uint8_t TwoWire::transmitting = 0;
 void (*TwoWire::user_onRequest)(void);
 void (*TwoWire::user_onReceive)(int);
 
-static int default_sda_pin = SDA;
-static int default_scl_pin = SCL;
```

Comment:
Now that these lines are removed, TwoWire::twi.sda and .scl are never initialized.